java/java.lsp.server/vscode/CHANGELOG.md

```diff
@@ -20,22 +20,37 @@
     under the License.
 
 -->
+
+## Version 12.4 Beta
+
+* Test results UI added using Test Explorer UI extension
+* Micronaut projects support added
+  * YAML configuration files code completion
+* Maven multi-project projects support
+* Maven repository index download is not performed
```

Comment:
I don't think this is so great achievement to be mentioned in a change log.

Reply:
Removed.